Source/Resources/Content/AzureManagement/MicrosoftAadIam/DiagnosticSettings/Logging.json

```diff
@@ -1,4 +1,5 @@
 {
+  "description":"What is it: Configures logging to capture Azure sign-in events, which can then be analyzed using Log Analytics Workspaces. \n Simeon Notes: Provides you with the data needed to monitor and be alerted on login activity within your tenant. Simeon's baseline captures all audit logs as well as all sign-in attempts. Logs can be viewed in the Analytics Workspace named 'tenant Name-M365logging' and are stored in the Azure Storage Account named tenantNameM365Logging. \n Learn More: https://docs.microsoft.com/en-us/azure/active-directory/reports-monitoring/howto-analyze-activity-logs-log-analytics",
```

Comment:
**What does this do?**
Configures logging to capture Azure sign-in events, which can then be analyzed using Log Analytics Workspaces. The baseline captures all audit logs as well as all sign-in attempts. Logs can be viewed in the Analytics Workspace named 'tenant Name-M365logging' and are stored in the Azure Storage Account named tenantNameM365Logging. 

**Why should you use this?**
Improves security by providing you with the data needed to monitor and be alerted of log in activity within your tenant.

**What is the end-user impact?**
N/A

**Learn more**
[Analyze Azure AD activity logs with Azure Monitor logs](https://docs.microsoft.com/en-us/azure/active-directory/reports-monitoring/howto-analyze-activity-logs-log-analytics)